src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -625,18 +625,16 @@ internal static List<AzureSqlSkuCategory> GetEligibleSkuCategories(string target
                     string jsonFile = File.ReadAllText(Path.Combine(assemblyPath, RecommendationConstants.DataFolder, RecommendationConstants.SqlVmCapability));
                     List<AzureSqlIaaSCapability> vmCapabilities = JsonConvert.DeserializeObject<List<AzureSqlIaaSCapability>>(jsonFile);
 
-                    if (includePreviewSkus)
+                    // Eb series capabilities stored separately 
+                    string computePreviewFilePath = Path.Combine(assemblyPath, RecommendationConstants.DataFolder, RecommendationConstants.SqlVmPreviewCapability);
+                    if (File.Exists(computePreviewFilePath))
                     {
-                        // Eb series (in preview) capabilities stored separately 
-                        string computePreviewFilePath = Path.Combine(assemblyPath, RecommendationConstants.DataFolder, RecommendationConstants.SqlVmPreviewCapability);
-                        if (File.Exists(computePreviewFilePath))
-                        {
-                            jsonFile = File.ReadAllText(computePreviewFilePath);
-                            List<AzureSqlIaaSCapability> vmPreviewCapabilities = JsonConvert.DeserializeObject<List<AzureSqlIaaSCapability>>(jsonFile);
+                        jsonFile = File.ReadAllText(computePreviewFilePath);
+                        List<AzureSqlIaaSCapability> vmPreviewCapabilities = JsonConvert.DeserializeObject<List<AzureSqlIaaSCapability>>(jsonFile);
```

Comment:
will these changes be backwards compatible?  or will the new nuget package be required?

Reply:
Previously, these specific VM SKUs (which are stored in a separate file) were only considered for recommendation if the preview flag was checked. Now that these SKUs are GAed, we consider them no matter what (hence removing the if statement). So for that change specifically, a new NuGet technically wasn't required since those SKUs already existed before this.

However, we do need a new NuGet because of changes made to the recommendation logic itself (which could how these SKUs even get recommended in the first place) as well as some other miscellaneous changes to SKU rec/assessment rules.